lib/utils/util.dart

```diff
@@ -55,6 +55,18 @@ class Util {
     return icon;
   }
 
+  static String getCurrencyCharacter({required Country country}) {
+    var currency = '€';
+    if (country == Country.us) {
+      currency = r'$';
+    }
+    if (Country.unitedKingdomCodes().contains(country.countryCode)) {
+      currency = '£';
+    }
+
+    return currency;
+  }
+
```

Comment:
I wouldn't do this. Me and Pavlo use the NumberFormat to get the symbol

Reply:
Thanks for the pointer ! 💪